src/components/donation/Steps/Submit/StripeCheckout/Paypal/Paypal.tsx

```diff
@@ -27,23 +22,12 @@ export default function PaypalCheckout(props: PaypalCheckoutStep) {
     splitLiq: liquidSplitPct.toString(),
   });
 
-  const dispatch = useSetter();
-
   return (
-    <Summary
-      classes={{
-        container: "flex flex-col isolate p-4 @md:p-8",
-        split: "mb-auto",
-      }}
-      Amount={currency(details.currency)}
-      amount={+details.amount}
-      splitLiq={liquidSplitPct}
-      onBack={() => dispatch(setStep("summary"))}
-    >
+    <>
       {isLoading ? (
-        <ContentLoader className="rounded h-14 w-full" />
+        <ContentLoader className="rounded h-10 w-40" />
       ) : isError || !orderId ? (
-        <Err error={error} />
+        <div className="rounded h-10 w-40 border border-red bg-red-l5" />
```

Comment:
This is how the UI looks when an error occurs:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/3396e333-f00e-4bf6-a38c-f59f8e9d678e)

The error should be displayed somehow, either as it was before this removal OR inside the error modal.

Reply:
Looks good to me, @SovereignAndrey is this ok?

Reply:
@ap-justin I would agree that if PayPal encounters an error, it is _a lot_ better UI/UX to remove the paypal button _and_ the OR separator from view. 

Having the red button with error there could make donors think that something is broken in general and that they can't donate (or they needlessly reach out to support asking about the error).

Reply:
> @ap-justin I would agree that if PayPal encounters an error, it is _a lot_ better UI/UX to remove the paypal button _and_ the OR separator from view.
> 
> Having the red button with error there could make donors think that something is broken in general and that they can't donate (or they needlessly reach out to support asking about the error).

@SovereignAndrey, applied hiding https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2745/commits/23b52b8e090d65aed417ac27f5d82cd838ec266d